### PR TITLE
Add homepage text, lighten links, add files for local CSS, update TRL…

### PR DESCRIPTION
**Pull request by seanaery**
…N Argon gem for fewer home facets. Closes TRLNDSC-3.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -53,7 +53,7 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'blacklight', "~> 6.7"
-gem 'trln_argon', git: 'https://github.com/trln/trln_argon'
+gem 'trln_argon', git: 'https://github.com/trln/trln_argon', tag: 'v0.5.16'
```
Comment (corylown) [inline Gemfile]: Sean, is there a reason this is limited to a single tagged release?

Comment (seanaery) [reply Gemfile]: @corylown Good question -- would it be better to `bundle update trln_argon` then commit files to the `.lock` file when updates from the engine are needed?

Comment (corylown) [reply Gemfile]: That's what I'd do (or `bundle update --source trln_argon`), unless there's some reason we don't want whatever the latest version of Argon is.

**Comment by corylown** [comment]
I'm seeing some style oddities with the location hierarchy facet that seems to have been introduced in this PR. Not quite sure why though.

This branch:
screen shot 2018-09-24 at 9 48 35 am

Master:
screen shot 2018-09-24 at 9 48 14 am

File: Gemfile (hunk 1 above)
Comment (seanaery) [reply Gemfile]: @corylown OK thanks -- will revise that shortly

**Comment by seanaery** [comment]
@corylown, @mikedaul -- The facet style oddities look to be coming from a recent commit on `trln_argon`. The present commit on `dul-argon-skin` only changes the link color.

**Comment by seanaery** [comment]
@corylown, @mikedaul -- Now that I've looked closer, looks like both apps have some style oddities. The missing arrow icons is a problem w/this PR; the chopped letter descenders (e.g., see the y characters on "Lilly" & "Divinity" ) is on `trln_argon`

**Comment by mikedaul** [comment]
@corylown - I'll fix the clipped descender issue upstream in trln-argon

**Comment by seanaery** [comment]
@corylown, @mikedaul -- I have revised the gem reference to `trln_argon`, plus the style import rule for the `blacklight_hierarchy` utility (that was causing the missing arrow images). Thanks for the feedback.

**Pull request merged**